src/compiler/transformers/component-lazy/lazy-constructor.ts

```diff
@@ -28,15 +29,13 @@ export const updateLazyComponentConstructor = (
 
     classMembers[cstrMethodIndex] = ts.factory.updateConstructorDeclaration(
       cstrMethod,
-      cstrMethod.decorators,
-      cstrMethod.modifiers,
+      retrieveTsModifiers(cstrMethod),
```

Comment:
It's worth calling out that a ctor doesn't receive decorators, which is why we use the helper for modifiers only here